stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -533,14 +541,14 @@ function getAccessTokenFromRefreshRequest(PasswordGrantConfig|DirectTokenConfig
     return doRequest(refreshUrl, refreshRequest, clientConfig, oauth2CacheEntry, clockSkewInSeconds);
 }
 
-# Execute the actual request and get the access token from authorization endpoint.
+# Executes the actual request and get the access token from authorization endpoint.
 #
 # + url - URL of the authorization endpoint
 # + request - Prepared request to be sent to the authorization endpoint
-# + clientConfig - HTTP client configurations which calls the authorization endpoint
+# + clientConfig - HTTP client configurations which is used to call the authorization endpoint
 # + oauth2CacheEntry - OAuth2 cache entry
 # + clockSkewInSeconds - Clock skew in seconds
-# + return - Access token received or `Error` if an error occurred during HTTP client invocation
+# + return - Received OAuth2 access token or `oauth2:Error` if an error occurred during HTTP client invocation
```

Comment:
```suggestion
# + return - Received OAuth2 access token or `oauth2:Error` if an error occurred during HTTP client invocation
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - Received OAuth2 access token or else `oauth2:Error` if an error occurred while HTTP client invocation
```